charts/artifact-hub/templates/_helpers.tpl

```diff
@@ -50,3 +50,13 @@ Selector labels
 app.kubernetes.io/name: {{ include "chart.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end -}}
+
+{{/*
+Kubernetes version
+Built-in object .Capabilities.KubeVersion.Minor can provide non-number output
+For example on GKE it returns "15+" instead of "15"
+*/}}
+{{- define "chart.KubernetesVersion" -}}
```

Comment:
Could we call this `KubectlImage` or something similar? `KubernetesVersion` is a bit misleading given it also includes the `bitnami/kubectl` part. I'd also be happy if you leave the `KubernetesVersion` name but it only contains the version, leaving the `bitnami/kubectl` part on the `hub_deployment` template. Whatever you like the most. I think it'll be more descriptive this way.

Reply:
Yep, it sounds reasonable. I've changed to store only Kubernetes version in KubernetesVersion.